recipes/stem/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "stem" %}
+{% set version = "1.6.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d7fe1fb13ed5a94d610b5ad77e9f1b3404db0ca0586ded7a34afd323e3b849ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://files.pythonhosted.org/packages/11/d5/e51983f81b38408ae2f0e166481ad867962f6fa07610fe71119534e12d41/stem-1.6.0.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - tox
+  run:
+    - python
+
+test:
+  imports:
+    - stem
+    - stem.descriptor
+    - stem.interpreter
+    - stem.response
+    - stem.util
+  commands:
```

Comment:
Is it necessary to test the existence of `tox` here?

Reply:
Hi @synapticarbors , how are you?

I made the required changes to the recipe but it seems that the during the test phase of the CI build, when running tox, the file tox.ini is not found but the tox.ini is in the distribution for stem.

Is there a way to point tox to right file?

Thanx.

SekouD.

Reply:
Sorry I missed this comment. I think you'd want:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-files

Let me know if you want to give that a try or if I should merge as is.